slo-workload/src/Utils.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace YdbPlatform\Ydb\Slo;
+
+use Exception;
+use Ydb\StatusIds\StatusCode;
+use YdbPlatform\Ydb\Traits\RequestTrait;
+use YdbPlatform\Ydb\Ydb;
+
+class Utils
+{
+    const MSG_TYPE = 1;
+    public static function initDriver(string $endpoint, string $db, string $process)
+    {
+        $endpointData = explode("://", $endpoint);
+        if (count($endpointData) != 2) throw new Exception("Invalid endpoint exception");
```

Comment:
Here and in other places in the PR:

Use brackets and dedicated block for throw.

```php
if (count($endpointData) != 2) {
    throw new Exception("Invalid endpoint exception");
}
```

It allow to use one style for all ifs and prevent see/write mistakes.